action.html

```diff
@@ -198,15 +198,15 @@ <h4>Some ready-made posts:</h4>
 						<p id="post-2">It’s <span class="red">#TimeFor5</span>, <span class="red">@DanaherCorp</span> and it’s time for you to release that audit! It’s been over a year since you promised to release one! ⏰</p>
 						<div style="margin-left: 5px;">
 							<button class="button copy" onclick="onclickButton(document.getElementById('post-2').textContent)">Copy text</button>
-							<a href="https://twitter.com/intent/tweet?text=It%E2%80%99s%20%23TimeFor5%2C%20%40DanaherCorp%20and%20it%E2%80%99s%20time%20for%20you%20to%20release%20that%20audit!%20It%E2%80%99s%20been%20over%20a%20year%20since%20you%20promised%20to%20release%20one!%20%E2%8F%B0%0Ahttps%3A%2F%2Fpic.twitter.com%2FodTkBe5TKr" target="_blank" class="button button-subtle">Click to tweet</a>
+							<a href="https://twitter.com/intent/tweet?text=.%40DanaherCorp%20promised%20to%20release%20an%20audit%20of%20the%20cost%20to%20produce%20their%20life-saving%20tests%2C%20but%20a%20year%20later%20they%20still%20haven%E2%80%99t%20released%20it!%20%23PeopleOverProfits%0Ahttps%3A%2F%2Fpic.twitter.com%2FZ4WgSTwBWx" target="_blank" class="button button-subtle">Click to tweet</a>
 						</div>
 					</div>
 
 
 					<div class="post">
 						<strong>Dave Green</strong>
 						<p class="red">@DaveGreen</p>
-						<p id="post-3" >It’s <span class="red">#TimeFor5</span>, <span class="red">@DanaherCorp</span> and it’s time for you to release that audit! It’s been over a year since you promised to release one! ⏰</p>
+						<p id="post-3" >.<span class="red">@DanaherCorp</span> promised to release an audit of the cost to produce their life-saving tests, but a year later they still haven’t released it! <span class="red">#PeopleOverProfits</span></p>
```

Comment:
Minor: This line appears to have a period at the beginning of the line. Was that intentional?

Reply:
Yup! Many twitter-style social media sites recognize posts with an @ tag at the beginning as replies, which restricts how much they're shown to other users